test/integration/consul-container/libs/consul-node/consul-container.go

```diff
@@ -0,0 +1,130 @@
+package consul_node
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/testcontainers/testcontainers-go/wait"
+
+	"github.com/docker/docker/pkg/ioutils"
+
+	"github.com/hashicorp/consul/integration/consul-container/libs/utils"
+
+	"github.com/testcontainers/testcontainers-go"
+
+	"github.com/hashicorp/consul/api"
+)
```

Comment:
nit: we generally organize our imports into three groups: stdlib, 3rd party, this module ([`goimports -local`](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) can do this for you automatically).

```suggestion
import (
	"context"
	"fmt"
	"os"
	"strings"
	"time"

	"github.com/docker/docker/pkg/ioutils"
	"github.com/testcontainers/testcontainers-go"
	"github.com/testcontainers/testcontainers-go/wait"
	"github.com/hashicorp/consul/api"

	"github.com/hashicorp/consul/integration/consul-container/libs/utils"
)
```